packages-exp/auth-exp/src/platform_browser/load_js.ts

```diff
@@ -25,7 +28,7 @@ export function _loadJS(url: string): Promise<Event> {
     const el = document.createElement('script');
     el.setAttribute('src', url);
     el.onload = resolve;
-    el.onerror = reject;
+    el.onerror = () => reject(_createError(AuthErrorCode.INTERNAL_ERROR));
```

Comment:
Should we pass the original error along?

Reply:
Done; we pass the original error if the field is populated, otherwise we create an error

Reply:
I think it's fine to always create a new error, but what I meant is - can we include the original error whether it's an error or a string as a field in the `INTERNAL_ERROR` object?

Reply:
Ah okay -- I just set the error event object as the customData field. That way we don't lose any info in the conversion